htdocs/projet/card.php

```diff
@@ -496,14 +496,11 @@
 			// Load new object
 			$newobject = new Project($db);
 			$newobject->fetch($result);
-			$newobject->fetch_optionals();
-			$newobject->fetch_thirdparty(); // Load new object
-			$object = $newobject;
-			$action = 'view';
```

Comment:
And what if you just set $action='edit' here to avoid the redirect ?

Reply:
@eldy it doens't work : the form is displayed but if you cancel or leave the page, the cloned project is not created at all.